docs/source/fitfunctions/CubicSpline.rst

```diff
@@ -9,22 +9,90 @@ CubicSpline
 Description
 -----------
 
-This function creates spline using the set of points and interpolates
-the input between them.
+This function creates spline of order 3 using the set of points and interpolates
+the input between them taking into account the least-squares fit
 
 First and second derivatives from the spline can be calculated by using
 the derivative1D function.
 
-CubicSpline function takes a set of attributes and a set of parameters.
-The first attrbiute is 'n' which has integer type and sets the number of
-interpolation points. The parameter names have the form 'yi' where 'y'
-is letter 'y' and 'i' is the parameter's index starting from 0 and have
-the type double. Likewise, the attribute names have the form 'xi'.
+A CubicSpline is a polynomial function :math:`f(x)` of order 3, defined between an interval :math:`a \leqslant x \leqslant b`.
+When using CubicSplines for interpolation or for fitting, we essentially chain CubicSplines together so that each
```

Comment:
Do you chain CubicSplines together or is it regular splines that are chained to make a Cubicspline?